pkg/http/http.go

```diff
@@ -188,13 +188,18 @@ func performRequest(req *http.Request, verifyTLS bool, params []queryParam) (int
 		return dialer.DialContext(ctx, network, addr)
 	}
 
+	proxyUrl, err := http.ProxyFromEnvironment(req)
+	if err == nil && proxyUrl != nil {
```

Comment:
If `err != nil`, should we log it? As it stands, in that scenario, we'll pass `nil` into `http.ProxyURL` which results in no proxy being used and the error will effectively be silenced

Reply:
Will add a debug log for this